sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -993,22 +1016,6 @@ def test_non_uniform_weights_toy_edge_case_reg():
         assert_greater(gb.predict([[1, 0]])[0], 0.5)
 
 
-def test_non_uniform_weights_toy_min_weight_leaf():
```

Comment:
It is removed because for trees built with gradient boosting in master, `tree.splitter` is a splitter object, but it's a string for all other trees built (and there are unit tests ensuring this is the desired behavior.) Since 'presort' is now an option, as opposed to indicated by passing in a hardcoded splitter object, `tree.splitter` is a string in gradient boosting. The call `tree.splitter.min_weight_leaf` thus doesn't hold anymore.